pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -615,6 +615,61 @@ mod season {
 			);
 		});
 	}
+
+	#[test]
+	fn active_season_hooks_should_update_active_season_on_block_initialize() {
+		ExtBuilder::default().organizer(ALICE).build().execute_with(|| {
+			assert!(AwesomeAvatars::active_season_id().is_none());
+
```

Comment:
We also need to check whether:
- `NextActiveSeasonId` is / isn't updated correctly
- `ActiveSeasonId` is `None` if out of season